packages/@okta/vuepress-site/use_cases/inline_hooks/registration_hook/registration_hook/index.md

```diff
@@ -0,0 +1,249 @@
+
+#### value
+
+The `value` object is the parameter to pass to the command.
+
+For `com.okta.user.profile.update` commands, `value` should be an object containing one or more name-value pairs for the attributes you wish to update, for example:
+
+```json
+{
+   "commands":[
+      {
+         "type":"com.okta.user.profile.update",
+         "value":{
+            "middleName":"Danger",
+            "customerId":12345
+         }
+      }
+   ]
+}
+```
+
+The above example assumes that there is an attribute `customerId` defined in your Okta user schema (`middleName` is defined by default).
+
+The same result could also be accomplished by means of two separate `com.okta.user.profile.update` commands, as follows:
+
+```json
+{
+   "commands":[
+      {
+         "type":"com.okta.user.profile.update",
+         "value":{
+            "middleName":"Danger"
+         }
+      },
+      {
+         "type":"com.okta.user.profile.update",
+         "value":{
+            "customerId":12345
+         }
+      }
+   ]
+}
+```
+
+For `com.okta.action.update` commands, `value` should be an object containing the attribute `action` set to a value of either `ALLOW` or `DENY`, indicating whether the registration should be permitted or not, for example:
+
+```json
+{
+   "commands":[
+      {
+         "type":"com.okta.action.update",
+         "value":{
+            "action":"DENY",
+         }
+      }
+   ]
+}
+```
+
+Registrations are allowed by default, so setting a value of `ALLOW` for the `action` field is valid but superfluous. 
+
+### error
+
+See [error](/use_cases/inline_hooks/) for general information on the structure to use for the `error` object.
+
+In the case of the Registration Inline Hook, the `error` object provides a way of displaying an error message to the end user who is trying to register. If you're using the Okta Sign-In Widget for Self-Service Registration, and have not customized its error handling behavior, what is displayed to the end user is the `errorSummary` of the first `ErrorCause` object that your external service returns.
+
+If you do not return any value for that `errorCauses` object, but deny the user's registration attempt via the `commands` object in your response to Okta, the following generic message is displayed to the end user: "Registration cannot be completed at this time".
+
+> Note: If you include an error object in your response, no commands will be executed and the registration will fail. This holds true even if the top-level `errorSummary` and the `errorCauses` objects are omitted.
+
+## Sample Listing of JSON Payload of Request
```

Comment:
Unless there's a doc convention around this, I would omit "listing" from the header and just call it something like **Sample Request Payload JSON** or **JSON for Sample Request Payload**. I would do likewise for the sample responses as well.

Reply:
Good point, sounds much better.